make/modules/jdk.jdwp.agent/Lib.gmk

```diff
@@ -47,6 +47,27 @@ TARGETS += $(BUILD_LIBDT_SOCKET)
 ## Build libjdwp
 ################################################################################
 
+# SapMachine 2024-02-16: Add dt_filesocket implementation
+$(eval $(call SetupJdkLibrary, BUILD_LIBDT_FILESOCKET, \
+    NAME := dt_filesocket, \
+    OPTIMIZATION := LOW, \
+    CFLAGS := $(CFLAGS_JDKLIB), \
+    DISABLED_WARNINGS_clang := format-nonliteral, \
+    EXTRA_HEADER_DIRS := \
+        include \
+        libjdwp/export, \
+    LDFLAGS := $(LDFLAGS_JDKLIB) \
+        $(call SET_SHARED_LIBRARY_ORIGIN), \
+    LIBS_windows := $(JKDLIB_LIBS) ws2_32.lib Advapi32.lib, \
```

Comment:
I'm sure you can remove $(JKDLIB_LIBS). Also not sure about Advapi32.lib ?

Reply:
This still needs to be checked.

Reply:
Advapi32 ist still needed for permission checking.